### DISCUSSION TITLE
I converted the whisper large v2 model to ggml 👾

### DISCUSSION THREAD

**Discussion by camenduru** [ggml-org/whisper.cpp#753] [Show and tell]
thanks to everyone ❤ for this cool project 🔥

https://huggingface.co/openai/whisper-large-v2
https://huggingface.co/4bit/whisper-large-v2-ggml/tree/main

```py
!git clone https://github.com/openai/whisper
!git clone https://github.com/ggerganov/whisper.cpp
!git clone https://huggingface.co/openai/whisper-large-v2
!pip install transformers
!python3 /workspace/whisper.cpp/models/convert-h5-to-ggml.py /workspace/whisper-large-v2/ /workspace/whisper .
```

**Comment by camenduru** [comment]
![FtlpQDJaUAAOwuN](https://user-images.githubusercontent.com/54370274/231736346-6f7dc875-2b25-40e8-a399-49a14916a507.jpg)
oops

**Reply by justarmadillo** [reply]
> ![FtlpQDJaUAAOwuN](https://user-images.githubusercontent.com/54370274/231736346-6f7dc875-2b25-40e8-a399-49a14916a507.jpg) oops

Thank you for your effort ;)
